src/glview/GLView.cc

```diff
@@ -131,9 +131,13 @@ void GLView::paintGL()
 
   if(bgcol != bgstopcol){
     glDisable(GL_DEPTH_TEST);
+
     glMatrixMode(GL_PROJECTION);
     glLoadIdentity();
 
+    glMatrixMode(GL_MODELVIEW);
+    glLoadIdentity();
```

Comment:
Is the `GL_PROJECTION` part still needed? It looks to me like that would do nothing?

Reply:
when I comment out `glMatrixMode(GL_PROJECTION);`, there is a square centered on the screen, which size depends on zoom level.

I followed https://community.khronos.org/t/gradient-background/54348

The rough idea seams to be to setup openGL that the corners of the view are a 1 unit away from center (independent of aspect ratio of the view), then look straight on, then look onto a square with gradient.

Reply:
Both ModelView and Projection matrices are combined for the overall transformation (sometimes referred to as MVP).  Loading the identity resets whichever matrix is currently specified by `glMatrixMode`, but both are used simultaneously on all vertices going through the fixed function pipeline.

Reply:
Ah, ok. Thanks.